docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<blockquote>within <a href="https://ade.maap-project.org/" target="_blank">the ? Algorithm Development Environment</a></blockquote><p>Need to ask JPL how to describe this ADE clearly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>